test/format/verify.c

```diff
@@ -137,12 +137,55 @@ table_mirror_fail_msg_flcs(WT_SESSION *session, const char *checkpoint, TABLE *b
       checkpoint ? " (checkpoint " : "", checkpoint ? checkpoint : "", checkpoint ? ")" : "");
 }
 
+/*
+ * position_cursor_before --
+ *     Place a cursor on the key directly preceding the target key. It's ok to return WT_NOTFOUND
+ *     from this function as any subsequent cursor next calls will begin at the start of the table.
+ */
+static int
+position_cursor_before(TABLE *table, WT_CURSOR *cursor, uint64_t target_keyno)
+{
+    WT_ITEM key;
+    int exact;
+    int ret;
```

Comment:
```suggestion
    WT_DECL_RET;
```
And we can remove `ret = 0;` below.

Reply:
Fixed in [d9fd6a8](https://github.com/wiredtiger/wiredtiger/pull/9660/commits/d9fd6a86fafaf9449bed957e42c84ccd3e114973)